class-es-wp-query-wrapper.php

```diff
@@ -1040,6 +1051,7 @@ public function get_posts() {
 		}
 
 		if ( 'ids' == $q['fields'] || 'id=>parent' == $q['fields'] ) {
+			$this->es_args['disable_cache'] = true;
```

Comment:
@philipjohn This seems to break things. I'm not familiar with this argument, so I'm guessing this is something wpcom-specific? If that's the case, and it's something necessary, this should be able to be added via the above filter `es_posts_request`. I'm going to remove this in #27.